plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoFailureRecoveryTest.java

```diff
@@ -102,16 +111,16 @@ protected void testRefreshMaterializedView()
     @Override
     protected void testUpdate()
     {
-        assertThatThrownBy(super::testUpdate).hasMessageContaining("This connector does not support modifying table rows");
-        abort("skipped");
+        // TODO: solve https://github.com/trinodb/trino/issues/22256
```

Comment:
Will file another pr for the FTE, the `$merge` op not support in transactional in Mongo